src/cascade/core/db.py

```diff
@@ -63,6 +65,18 @@ def __dir__(self):
 save_results = ModuleProxy("save_results")
 
 
+def use_local_odbc_ini():
+    """The password vault is an odbc.ini file that's on a drive that
+    isn't accessible from all nodes, so copy it to a place it
+    can be found and point the connection generation to it."""
+    have_default = Path("/home/j").exists() or Path("~/.odbc.ini").expanduser().exists()
+    if LOCAL_ODBC.exists() and not have_default:
+        CODELOG.info(f"Using odbc.ini at {LOCAL_ODBC}")
+        db_tools.config.DBConfig(odbc_filepath=str(LOCAL_ODBC))
+    else:
+        CODELOG.debug(f"Using default odbc.ini")
```

Comment:
Should there be a third branch here where we blow up if there's no default and also no J? It'll happen on it's own later but... hm, no, you really do want to lazily fail because you might be doing something that doesn't require the databases. Ok. Never mind.